### PR TITLE
Fix OSC 888 after qml merge

### PR DESCRIPTION
**Pull request by Yaraslaut**
Update  OSC 888 implementation to handle QML

### CODE REVIEW

File: src/contour/display/TerminalWidget.cpp [hunk 1]
```diff
@@ -684,6 +684,11 @@ void TerminalWidget::paint()
 
         terminal().tick(steady_clock::now());
         _renderer->render(terminal(), _renderingPressure);
+        if(_doDumpState)
```
Comment (christianparpart) [inline src/contour/display/TerminalWidget.cpp:687]:
```suggestion
        if (_doDumpState)
```
Didn't clang-format check catch this? hmmm 🤔

File: src/contour/display/TerminalWidget.cpp [hunk 2]
```diff
@@ -996,6 +1001,12 @@ void TerminalWidget::inspect()
 
 void TerminalWidget::doDumpState()
 {
+    _doDumpState = true;
+}
+
+void TerminalWidget::doDumpStateInternal()
+{
+
```
Comment (christianparpart) [inline src/contour/display/TerminalWidget.cpp:1009]:
```suggestion
```

File: src/contour/display/TerminalWidget.cpp [hunk 3]
```diff
@@ -1012,6 +1023,7 @@ void TerminalWidget::doDumpState()
         return;
     }
 
+
```
Comment (christianparpart) [inline src/contour/display/TerminalWidget.cpp:1026]:
```suggestion
```

**Review by christianparpart — COMMENTED** [review]
LGTM, but I feel like the only things that didn't look right are things that clang-format should have cought. 🤔 ...

**Comment by Yaraslaut** [comment]
> LGTM, but I feel like the only things that didn't look right are things that clang-format should have cought. 🤔 ...

I also realized that qml image grab does not grab OpenGL part but only qml itself, so I need to update it a bit

**Review by christianparpart — DISMISSED** [review]
Many thanks. :)

**Pull request merged**